index.js

```diff
@@ -199,7 +199,11 @@ function hash(src, filePath) {
 }
 
 function symlinkOrCopyFromCache(entry, dest, relativePath) {
-  mkdirp.sync(path.dirname(entry.outputFile));
-
-  symlinkOrCopySync(entry.cacheFile, dest + '/' + relativePath);
+  try {
```

Comment:
i don't know if persistent-filter needs this, as the whole TreeDiff + patch stuff should handle this for us.